### PR TITLE
Add mentions to DHAPI

### PR DESCRIPTION
**Pull request by Dominilk01**
I made this affter i saw [this](https://discord.com/channels/850844430830534696/1008710839173382256/1267877334502084649) inside DH discord

### CODE REVIEW

File: src/main/java/eu/decentsoftware/holograms/api/holograms/HologramPage.java [hunk 1]
```diff
@@ -166,6 +167,7 @@ public boolean addLine(@NonNull HologramLine line) {
     /**
      * Insert a new line into this hologram page.
      *
+     * @see eu.decentsoftware.holograms.api.DHAPI#insertHologramLine(Hologram, int, String)
      * @param index Index of the new line.
      * @param line  New line.
      * @return Boolean whether the operation was successful.
```
Comment (Andre601) [inline src/main/java/eu/decentsoftware/holograms/api/holograms/HologramPage.java:173]:
```suggestion
     * @param index Index of the new line.
     * @param line  New line.
     * @return Boolean whether the operation was successful.
     * @see eu.decentsoftware.holograms.api.DHAPI#insertHologramLine(Hologram, int, String)
```

File: src/main/java/eu/decentsoftware/holograms/api/holograms/HologramPage.java [hunk 2]
```diff
@@ -183,6 +185,7 @@ public boolean insertLine(int index, @NonNull HologramLine line) {
     /**
      * Set new content of a line in this hologram page.
      *
+     * @see eu.decentsoftware.holograms.api.DHAPI#setHologramLine(HologramPage, int, String)
      * @param index   Index of the line.
      * @param content Line's new content.
      * @return Boolean whether the operation was successful.
```
Comment (Andre601) [inline src/main/java/eu/decentsoftware/holograms/api/holograms/HologramPage.java:191]:
```suggestion
     * @param index   Index of the line.
     * @param content Line's new content.
     * @return Boolean whether the operation was successful.
     * @see eu.decentsoftware.holograms.api.DHAPI#setHologramLine(HologramPage, int, String)
```

File: src/main/java/eu/decentsoftware/holograms/api/holograms/HologramPage.java [hunk 3]
```diff
@@ -208,6 +211,7 @@ public boolean setLine(int index, @NonNull String content) {
     /**
      * Get line on a specified index in this hologram page.
      *
+     * @see eu.decentsoftware.holograms.api.DHAPI#getHologramLine(HologramPage, int)
      * @param index Index of the line.
      * @return The HologramLine or null if it wasn't found.
```
Comment (Andre601) [inline src/main/java/eu/decentsoftware/holograms/api/holograms/HologramPage.java:216]:
```suggestion
     * @param index Index of the line.
     * @return The HologramLine or null if it wasn't found.
     * @see eu.decentsoftware.holograms.api.DHAPI#getHologramLine(HologramPage, int)
```

File: src/main/java/eu/decentsoftware/holograms/api/holograms/HologramPage.java [hunk 4]
```diff
@@ -221,6 +225,7 @@ public HologramLine getLine(int index) {
     /**
      * Remove a line from this hologram page.
      *
+     * @see eu.decentsoftware.holograms.api.DHAPI#removeHologramLine(HologramPage, int)
      * @param index Index of the line.
      * @return The removed line or null if it wasn't found.
```
Comment (Andre601) [inline src/main/java/eu/decentsoftware/holograms/api/holograms/HologramPage.java:230]:
```suggestion
     * @param index Index of the line.
     * @return The removed line or null if it wasn't found.
     * @see eu.decentsoftware.holograms.api.DHAPI#removeHologramLine(HologramPage, int)
```

**Review by Andre601 — CHANGES_REQUESTED** [review]
See tags should come after param and return tags.

**Comment by Andre601** [comment]
It's about readability in the source and following common Conventions for Javadoc comments. See [How to Write Doc Comments for the Javadoc Tool](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html), especially the [Format of a Doc Comment](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#format) Section.

In all honesty am I not even sure if this would properly compile in Javadocs, if someone ever wanted to generate those, because iirc does the JD comment require a description to be set. I can be wrong here tho.

File: src/main/java/eu/decentsoftware/holograms/api/holograms/Hologram.java [hunk 5]
```diff
@@ -533,15 +541,6 @@ public void onQuit(@NonNull Player player) {
...
 
-    /**
-     * Set default display state
-     *
-     * @param state state
-     */
-    public void setDefaultVisibleState(boolean state) {
-        this.defaultVisibleState = state;
-    }
-
```
[the remark this answers is not in the document]
Comment (Dominilk01) [reply src/main/java/eu/decentsoftware/holograms/api/holograms/Hologram.java:544]: Lombok alredy generate

**Pull request merged**